packages/lib-classifier/src/store/SubjectViewerStore/SubjectViewerStore.js

```diff
@@ -2,6 +2,16 @@ import asyncStates from '@zooniverse/async-states'
 import { autorun } from 'mobx'
 import { addDisposer, getRoot, isValidReference, types } from 'mobx-state-tree'
 
+const NO_IMAGE_TOOLBAR_MIME_TYPES = [
+  'audio/mp3',
+  'audio/m4a',
+  'audio/mpeg',
+  'text/plain',
```

Comment:
This will remove the zoom, and light curve marking controls, from Planet Hunters.

Reply:
Ideally, PH-TESS would use JSON subjects but their subjects are `.txt` files.